src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -133,6 +134,25 @@ internal static SafeSharedX509NameStackHandle SslGetClientCAList(SafeSslHandle s
             return handle;
         }
 
+        internal static bool AddExtraChainCertificates(SafeSslHandle sslContext, X509Chain chain)
+        {
+            Debug.Assert(chain != null, "chain != null");
```

Comment:
Yeah, because I followed SetSslCertificate's Assert example at the time to stay consistent. It's not predictable what the expectation is for asset messages based on your code base and it's not in the coding-style outline. On further perusal, assert messages are all over the place in different files, some with descriptive messages (what I would normally write), some with the exact code being run (as above), some with the condition expected, some no message at all. Might be an idea to decide on a style and start using it, if the contents of assert statements are considered important.